cmake/alpakaCommon.cmake

```diff
@@ -520,6 +523,138 @@ if(ALPAKA_ACC_GPU_HIP_ENABLE)
 
 endif() # HIP
 
+#-------------------------------------------------------------------------------
+# SYCL settings
+if(ALPAKA_ACC_SYCL_ENABLE)
+    # Possible SYCL platforms
+    cmake_dependent_option(ALPAKA_SYCL_PLATFORM_ONEAPI "Enable Intel oneAPI platform for the SYCL back-end" OFF "ALPAKA_ACC_SYCL_ENABLE" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_PLATFORM_XILINX "Enable Xilinx platform for the SYCL back-end" OFF "ALPAKA_ACC_SYCL_ENABLE" OFF)
+    # Possible oneAPI targets
+    cmake_dependent_option(ALPAKA_SYCL_ONEAPI_CPU "Enable oneAPI CPU targets for the SYCL back-end" OFF "ALPAKA_SYCL_PLATFORM_ONEAPI" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_ONEAPI_FPGA "Enable oneAPI FPGA targets for the SYCL back-end" OFF "ALPAKA_SYCL_PLATFORM_ONEAPI" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_ONEAPI_GPU "Enable oneAPI GPU targets for the SYCL back-end" OFF "ALPAKA_SYCL_PLATFORM_ONEAPI" OFF)
+    # Intel FPGA emulation / simulation
+    cmake_dependent_option(ALPAKA_SYCL_ONEAPI_FPGA_EMULATION "Enable oneAPI FPGA emulator" ON "ALPAKA_SYCL_ONEAPI_FPGA" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_ONEAPI_FPGA_SIMULATION "Enable oneAPI FPGA simulator" OFF "ALPAKA_SYCL_ONEAPI_FPGA" OFF)
+    # Xilinx FPGA targets
+    cmake_dependent_option(ALPAKA_SYCL_XILINX_FPGA_HARDWARE_EMULATION "Enable Xilinx FPGA hardware emulation" ON "ALPAKA_SYCL_PLATFORM_XILINX" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_XILINX_FPGA_HARDWARE "Enable Xilinx FPGA hardware synthesis" OFF "ALPAKA_SYCL_PLATFORM_XILINX" OFF)
+
+    # Enable device-side printing to stdout
+    cmake_dependent_option(ALPAKA_SYCL_ENABLE_STREAM "Enable device-side printing to stdout" OFF "ALPAKA_ACC_SYCL_ENABLE" OFF)
+    if(BUILD_TESTING)
+        set(ALPAKA_SYCL_ENABLE_STREAM ON CACHE BOOL "Enable device-side printing to stdout" FORCE)
+    endif()
```

Comment:
I don't think the name `ALPAKA_SYCL_ENABLE_STREAM` is good. I thought this is something similar to CUDA streams. Is this only related to device side printing? Maybe call it `ALPAKA_SYCL_ENABLE_DEVICE_SIDE_PRINTING` then.

Reply:
>  Is this only related to device side printing?

Yup, SYCL basically has a (strongly reduced) iostream.